syntax = "proto2";

package interface;

// Next id: 3
message WavParams {
  optional int32 sample_rate = 1;
  optional int32 bit_depth = 2;
}